iiod/responder.c

```diff
@@ -445,6 +458,8 @@ static void handle_create_buffer(struct parser_pdata *pdata,
 			entry->words[BIT_WORD(i)] &= ~BIT_MASK(i);
 	}
 
+	entry->is_tx = iio_buffer_is_tx(buf);
```

Comment:
It is used in the same commit, though. The value is just cached to avoid calling `iio_buffer_is_tx` every time.

I guess this could be cached inside the `iio_buffer` structure instead, yeah.